docker/php/fpm-php.ini

```diff
@@ -0,0 +1,4 @@
+memory_limit = -1
```

Comment:
Não é um valor aceitável, pode dar crash.

Reply:
Então @edersoares, como mencionado no tutorial, o docker não é recomendado para produção, por isso não vi problema definir tal valor para ambiente de desenvolvimento tendo em vista que não tem uma sobrecarga de processamento.

Podemos definir um valor de 2GB se for o caso!

Em um ambiente de produção sim, é um valor que não pode ser utilizado.